node/src/components/contract_runtime.rs

```diff
@@ -238,7 +239,7 @@ where
 impl ContractRuntime {
     /// How many blocks are backed up in the queue
     pub(crate) fn queue_depth(&self) -> usize {
-        self.exec_queue.lock().unwrap().len()
+        self.exec_queue.lock().expect("mutex poisoned").len()
```

Comment:
+1 for changing to `expect()`, but to make it even more worthwhile, it'd be nice to have a unique error message for each call.

Reply:
added more suggestive error messages here: https://github.com/casper-network/casper-node/pull/3471/commits/e0e7449b61d8f0d78e507b31303941aaa2335ac6